packages/react-popover/src/utils.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+// Need to disable compilation for aliases: https://github.com/microsoft/fluentui/pull/16976/files#r575447074
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Button as ReactButton, ButtonProps } from '@fluentui/react-button';
+
+export const ExampleContent = () => {
+  return (
+    <div>
+      <h3>Popover content</h3>
+
+      <div>This is some popover content</div>
+    </div>
+  );
+};
+
+export const Button = React.forwardRef<HTMLElement, ButtonProps>((props, ref) => <ReactButton {...props} ref={ref} />);
```

Comment:
Can you please clarify why this is required? Why not re-export or using directly?

Reply:
will reexport, I thought at some point that ts-ignoring once would be a good idea... now knowing how codesandbox will work .... it isn't a good idea 

will revert